apps/docs/scripts/search/generate-embeddings.ts

```diff
@@ -185,6 +197,7 @@ async function generateEmbeddings() {
               content,
               token_count: embeddingResponse.usage.total_tokens,
               embedding: responseData.embedding,
+              hf_embedding: featureArray,
```

Comment:
I wonder if it would be better to name embedding columns by their model vs platform? Could give us more flexibility in the future. What do you think?